### PR TITLE
Use bzlmod internally

### PR DESCRIPTION
**Pull request by cpsauer**
Forms the basis for releasing with bzlmod in the future

### CODE REVIEW

File: MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module(
+    name = "google_toolbox_for_mac",
+    version = "3.0.0",
```
Comment (thomasvl) [inline MODULE.bazel:3]: I assume we'll need to update this when we do releases and bump the version info in cocoapods?

Comment (cpsauer) [reply MODULE.bazel:3]: Yes, indeed; that's my understanding.

**Review by thomasvl — APPROVED** [review]
Thanks!

**Comment by cpsauer** [comment]
Thanks for being so great to work with, Thomas :)

**Pull request merged**